cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -145,14 +145,14 @@ def get_splice_junctions_bed(self, case_id: str, sample_id: str) -> Optional[Fil
         return splice_junctions_bed
 
     def get_rna_coverage_bigwig(self, case_id: str, sample_id: str) -> Optional[File]:
-        """Return a RNA coverage bigwig file for case in housekeeper."""
+        """Return an RNA coverage bigwig file for case in Housekeeper."""
 
-        tags: Set[str] = {"coverage", "bigwig", sample_id}
+        tags: List[str] = ["coverage", "bigwig", sample_id]
```

Comment:
So is this supposed to be a list or a set?

Reply:
Changed now :)